packages/editor/src/client/api/comment.graphql

```diff
@@ -0,0 +1,71 @@
+fragment CommentRef on Comment {
```

Comment:
i would distinguish between `CommentRef` (id, state, revisions) and `FullComment` (everything)

Reply:
@roosnic1  When to use CommentRef?

Reply:
maybe not now but for the future.